string     target_arm
float64    target_x
float64    target_y
float64    target_z
---
bool       success
float64[]  joints

